.Lib9c.Plugin.Shared/IPluginActionEvaluator.cs

```diff
@@ -1,7 +1,14 @@
+using Libplanet.Action;
+using Libplanet.Action.Loader;
+
 namespace Lib9c.Plugin.Shared
 {
     public interface IPluginActionEvaluator
     {
         byte[][] Evaluate(byte[] blockBytes, byte[]? baseStateRootHashBytes);
+
+        IActionLoader ActionLoader { get; }
+
+        IPolicyActionsRegistry PolicyActionsRegistry { get; }
```

Comment:
I think the `IActionLoader` and `IPolicyActionsRegistry` types are too detailed types to include in the plugin system.

Please check the following types will never be changed? Or Libplanet will care type backward-compatibility for Lib9c pluggable system?

 - `IActionLoader`
 - `IPolicyActionsRegistry`
 - `IAction`
   - `IActionContext`
     - `IWorld`
       - `IWorldDelta`
       - `IAccount`
     - `ITransaction`
     - `EvidenceBase`
     - `IRandom`
     - `Address`
     - `TxId`